app/src/components/superheroes/SuperheroList.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <ul v-if="superheroes">
+    <li v-for="superhero in superheroes" :key="superhero.name">
```

Comment:
```suggestion
    <li v-for="superhero in superheroes" :key="superhero.id">
```
If you loop on a name, you run into a duplicate key error. Try to loop on id or a unique index.